src/middlewared/middlewared/plugins/enclosure.py

```diff
@@ -98,8 +98,11 @@ def query(self, filters, options):
                         "descriptor": elem.descriptor,
                         "status": elem.status,
                         "value": elem.value,
-                        "value_raw": hex(elem.value_raw),
+                        "value_raw": 0x0,
                     }
+                    if isinstance(elem.value_raw, int):
+                        element["value_raw"] = hex(elem.value_raw)
```

Comment:
Might be an idea to add a comment explaining **why** we could get a non-int as the `value_raw`.